source/MRViewer/MRRibbonMenu.cpp

```diff
@@ -998,6 +998,39 @@ void RibbonMenu::cloneTree( const std::vector<std::shared_ptr<Object>>& selected
     }
 }
 
+void RibbonMenu::cloneSelectedPart( const std::shared_ptr<Object>& object )
```

Comment:
Copied from `SelectionToNewObject::onEnable_()`.
Proposed changes to `SelectionToNewObject::onEnable_()`:
``
     auto selected = getAllObjectsInTree<VisualObject>( &SceneRoot::get(), ObjectSelectivityType::Selected )[0];
- <copied part>
+   MR::RibbonMenu::cloneSelectedPart( selected );
    return false;
``